compiler/luci-interpreter/pal/linux/KernelsToBuild.lst

```diff
@@ -0,0 +1,67 @@
+REGISTER_KERNEL(Add)
```

Comment:
Do we need a license text here?

Reply:
I don't think so - this is not source code